.github/workflows/tests.yml

```diff
@@ -29,18 +29,25 @@ jobs:
       PGUSER: discourse
       PGPASSWORD: discourse
       USES_PARALLEL_DATABASES: ${{ matrix.build_type == 'backend' && matrix.target == 'core' }}
+      EMBER_CLI_PLUGIN_ASSETS: ${{ matrix.ember_cli_plugin_assets }}
 
     strategy:
       fail-fast: false
 
       matrix:
         build_type: [backend, frontend, annotations]
         target: [core, plugins]
+        ember_cli_plugin_assets: ['1', '0']
```

Comment:
For how long do we want to carry both implementations? N+1.beta1? 😃

Reply:
I am thinking "less than one week", as long as there are no surprises when rolling out